server/_dbConnect.ts

```diff
@@ -17,7 +17,6 @@ export const dbConnect = async (): Promise<MongoClient | undefined> => {
     await client.connect()
     // Establish and verify connection
     await client.db('admin').command({ ping: 1 })
-    console.log('Connected successfully to server')
     return client
```

Comment:
We really should add caching for this, unless it's being done in a different branch or was implemented somehwere else:
https://stackoverflow.com/questions/24621940/how-to-properly-reuse-connection-to-mongodb-across-nodejs-application-and-module